rep-2000.rst

```diff
@@ -341,56 +345,59 @@ Middleware Implementation Support:
-+-------------+----------------+---------------+----------------+---------------------+
++-------------+-------------------------------------------------+------------------------------------+
+|             |                 Required Support                |        Recommended Support         |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Package     | Ubuntu  Bionic |     MacOS**   |   Windows 10** | Debian Stretch | OpenEmbedded Thud |
++=============+================+===============+================+================+===================+
+| CMake       |     3.10.2     |     3.14.4    |     3.14.4     |      3.7.2     |       3.12.2      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| EmPY        |                                      3.3.2                                           |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Ogre        |                                      1.10*                       |        N/A        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenCV      |     3.2.0      |     4.1.0     |     3.4.6*     |      3.2*      |        3.2        |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| OpenSSL     |     1.1.0g     |     1.0.2r    |     1.0.2r     |      1.1.0j    |        1.0.2q     |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Poco        |     1.8.0      |     1.9.0     |     1.8.0*     |      1.8.0*    |        1.8.0      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Python      |     3.6.5      |     3.7.3     |     3.7.3      |      3.5.3     |        3.5.2      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Qt          |     5.9.5      |     5.12.3    |     5.10.0     |      5.7.1     |        5.9.8      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+|                              |         **Linux only**         |                                    |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| PCL         |     1.8.1      |     N/A       |     N/A        |      1.8.0     |        1.8.1      |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+|                                **RMW DDS Milddleware Providers**                                   |
++-------------+----------------+---------------+----------------+----------------+-------------------+
+| Connext DDS |                      5.3.1                      |       N/A      |         N/A       |
```

Comment:
Please merge the two adjacent "N/A" cells.

Reply:
Done, thanks!